### ISSUE TITLE
LP dashboard and website homepage tiles are showing terminated markets

### ISSUE THREAD

**Issue by JonRay15** [vegaprotocol/vega.xyz#721]
## Issue 

Mainnet [dashboard](https://vega.xyz/liquidity-provision) is displaying the markets that were terminated this morning. 

![Image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/3067fec2-9523-4106-a9da-6e5a61b953a1)

All three are showing as "Continuous" for some reason. 

In Console they appear with a status of "Settled"  .. we should be detecting this same status and if settled should be removing them from the view completely, I see no reason to show them here. 

![Image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/3d701dcf-1551-4a87-b93d-c58a83c3f5ab)

Once this LP dashboard change is done we can revisit whether additional change is needed on the homepage to make it stop showing them too.

## Proposal

Add a filter to the query on the markets API to exclude any records where `state` is one of: `STATE_REJECTED`, `STATE_CANCELLED`, `STATE_CLOSED`, `STATE_SETTLED`, `STATE_PROPOSED`.

For reference, this means we are including the following: `STATE_UNSPECIFIED`, `STATE_PENDING`, `STATE_ACTIVE`, `STATE_SUSPENDED`, `STATE_TRADING_TERMINATED`.

I have opted to only exclude the most "final" states of "not a market anymore" so we're erring on the side of including where there's doubt.  The only one that is contentious I think is trading terminated.  We could exclude this too since a terminated market does not need liquidity, but generally speaking markets only stay in that state for a few hours, and I would normally expect a market maker to not pull their liquidity until they have settled the market, so I think its best to include.

**Comment by ciaran-** [comment]
@JonRay15 PR incoming for this. Do we want to include `state` as a column in the table as well, or are we happy filtering by it silently ?

**Comment by JonRay15** [comment]
I think that adding state is just going to confuse people... 

cos you can have all the below, which can exist I believe in various combinations that I dont dare to attempt to list out! 

![image](https://github.com/vegaprotocol/vega.xyz/assets/89918817/562de512-963e-447c-b610-07d034119f23)

Ultimately for the LP Dashboard we care about:

- Market in opening auction - this has a trading mode, would be in state pending or proposed, cant remember, but neither adds much useful info 
- Market in continuous trading - this is a trading mode, state would be active, which is obvious 
- Market in auction - this is a trading mode, irritatingly I dont know if we know the diff between liquidity auction vs price monitoring auction but I seem to recall we do ... in either case state here is suspended, which again doesnt really add much
- Market terminated - I included this above and I guess it would show `TRADING_MODE_NO_TRADING` ... here the state does add some useful info but its the only case.  I am actually now thinking for simplicity lets not show terminated ones cos `TRADING_MODE_NO_TRADING` will raise questions for the few hours it is the case

So TLDR 
1. No dont add state 
2. Can you exclude state = `STATE_TRADING_TERMINATED` on second thought

**Comment by daunatv** [comment]
Should exclude state =` STATE_SETTLED` as well.
Think the main issue right now is that the markets are shown as continuous and that they are taking up 2 out of 4 spots on the main page.
<img width="1479" alt="Screenshot 2023-07-08 at 22 05 01" src="https://github.com/vegaprotocol/vega.xyz/assets/100769129/4ed25e65-ff29-4f67-84d8-5f7c5d7fbbcf">

**Comment by JonRay15** [comment]
This was subsequently amended to also exclude `STATE_PROPOSED`
